apps/pdal.cpp

```diff
@@ -46,6 +46,13 @@
 #include <string>
```

Comment:
General: is there some reason not to wait until the log level has been extracted in execute() to install the handler?